core/src/test/java/org/apache/iceberg/TestMetricUtil.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * Utils for testing Metrics.
+ */
+public class TestMetricUtil {
+
+  private TestMetricUtil() {
+  }
+
+  /**
+   * @param properties table properties
+   * @param sortOrder table sort order
+   * @param schema table schema
+   * @return table
+   */
+  public static Table createTestTable(Map<String, String> properties, SortOrder sortOrder,
```

Comment:
What about calling `table.replaceSortOrder()` within tests and using the existing `createTable`?

Reply:
Yep that's cleaner, done